Maya/Samples/glTF 2.0/BabylonMaterialAttributes/MaxSimultaneousLights.gltf

```diff
@@ -0,0 +1 @@
e":"VEC4","name":"accessorTangents"},{"bufferView":1,"byteOffset":68484,"componentType":5126,"count":439,"type":"VEC3","name":"accessorNormals"},{"bufferView":3,"byteOffset":21072,"componentType":5126,"count":439,"type":"VEC2","name":"accessorUVs"},{"bufferView":4,"componentType":5126,"count":2,"max":[8.291667],"min":[0.0],"type":"SCALAR","name":"accessorAnimationInput"},{"bufferView":5,"componentType":5126,"count":2,"type":"VEC4","name":"accessorAnimationRotations"},{"bufferView":4,"byteOffset":8,"componentType":5126,"count":2,"max":[8.291667],"min":[0.0],"type":"SCALAR","name":"accessorAnimationInput"},{"bufferView":6,"componentType":5126,"count":2,"type":"VEC3","name":"accessorAnimationPositions"}],"bufferViews":[{"buffer":0,"byteLength":31920,"name":"bufferViewScalar"},{"buffer":0,"byteOffset":31920,"byteLength":73752,"byteStride":12,"name":"bufferViewFloatVec3"},{"buffer":0,"byteOffset":105672,"byteLength":49168,"byteStride":16,"name":"bufferViewFloatVec4"},{"buffer":0,"byteOffset":154840,"byteLength":24584,"byteStride":8,"name":"bufferViewFloatVec2"},{"buffer":0,"byteOffset":179424,"byteLength":16,"name":"bufferViewAnimationFloatScalar"},{"buffer":0,"byteOffset":179440,"byteLength":32,"name":"bufferViewAnimationFloatVec4"},{"buffer":0,"byteOffset":179472,"byteLength":24,"name":"bufferViewAnimationFloatVec3"}],"buffers":[{"uri":"MaxSimultaneousLights.bin","byteLength":179496}],"materials":[{"pbrMetallicRoughness":{"baseColorFactor":[0.5,0.0,0.0,1.0],"metallicFactor":0.0,"roughnessFactor":0.2581639},"emissiveFactor":[0.0,0.0,0.0],"alphaMode":"OPAQUE","name":"phong1"},{"pbrMetallicRoughness":{"baseColorFactor":[0.0,1.0,0.0,1.0],"metallicFactor":0.0,"roughnessFactor":0.33},"emissiveFactor":[0.0,0.0,0.0],"alphaMode":"OPAQUE","name":"StingrayPBS1"},{"pbrMetallicRoughness":{"baseColorFactor":[0.0,0.0,0.8,1.0],"metallicFactor":0.0,"roughnessFactor":0.1},"emissiveFactor":[0.0,0.0,0.0],"alphaMode":"OPAQUE","name":"aiStandardSurface1"},{"pbrMetallicRoughness":{"baseColorFactor":[0.25,0.25,0.25,1.0],"metallicFactor":0.0,"roughnessFactor":0.450053632},"emissiveFactor":[0.0,0.0,0.0],"alphaMode":"OPAQUE","name":"lambert1"},{"pbrMetallicRoughness":{"baseColorFactor":[0.0,0.0,0.8,1.0],"metallicFactor":0.0,"roughnessFactor":0.1},"emissiveFactor":[0.0,0.0,0.0],"alphaMode":"OPAQUE","name":"pasted__aiStandardSurface1"},{"pbrMetallicRoughness":{"baseColorFactor":[0.5,0.0,0.0,1.0],"metallicFactor":0.0,"roughnessFactor":0.2581639},"emissiveFactor":[0.0,0.0,0.0],"alphaMode":"OPAQUE","name":"pasted__phong1"},{"pbrMetallicRoughness":{"baseColorFactor":[0.0,1.0,0.0,1.0],"metallicFactor":0.0,"roughnessFactor":0.33},"emissiveFactor":[0.0,0.0,0.0],"alphaMode":"OPAQUE","name":"pasted__StingrayPBS1"}],"animations":[{"channels":[{"sampler":0,"target":{"node":11,"path":"rotation"}},{"sampler":1,"target":{"node":11,"path":"translation"}}],"samplers":[{"input":35,"interpolation":"LINEAR","output":36},{"input":37,"interpolation":"LINEAR","output":38}],"name":"All Animations"}]}
```

Comment:
one issue in this file: `"scale":null`

Reply:
I think I already addressed this issue via #645, @noalak, this should be resolved after merging master